config/olm/kubernetes/0.1.0/bundle/manifests/dynatrace-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,395 @@
+    mediatype: image/png
+  install:
+    spec:
+      clusterPermissions:
+      - rules:
+        - apiGroups:
+          - ""
+          resourceNames:
+          - kube-system
+          resources:
+          - namespaces
+          verbs:
+          - get
+          - list
+          - watch
+        serviceAccountName: dynatrace-operator
+      deployments:
+      - name: dynatrace-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: dynatrace-operator
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                dynatrace: operator
+                name: dynatrace-operator
+                operator: activegate
+            spec:
+              containers:
+              - env:
+                - name: POD_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.namespace
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                image: dynatrace-operator:latest
```

Comment:
This is wrong. This should be `quay.io/dynatrace/dynatrace-operator:v0.1.0`

Please also use this same Quay image as containerImage when releasing it to community-operators.

Reply:
Updated